README.md

```diff
@@ -80,6 +80,10 @@ Pass an input element into a new instance of `Ca11y`, and optionally pass in pro
     'Thirtieth',
     'Thirty First'
   ],
+  toggle: {
+    html: '📅',
```

Comment:
@greypants Not sure what your plans are for actual defaults for this lib. Right now there are SVGs icons as the defaults here - is that what we plan to ship? (no strong feels here, just need README to reflect.)